flutter/lib/src/native/cocoa/binding.dart

```diff
@@ -31229,13 +31229,13 @@ class NSObject extends _ObjCWrapper {
   }
 }
 
-class ObjCSel extends ffi.Opaque {}
+base class ObjCSel extends ffi.Opaque {}
```

Comment:
these should be `final class`, see https://github.com/getsentry/sentry-dart/blob/9da696df2d576d3a2ed069249da7eaed5441c653/flutter/scripts/generate-cocoa-bindings.sh#L28

Reply:
Also can you please update the script?